app/views/schools/participants/_currently_training_sorted_by_induction_start_date.html.erb

```diff
@@ -0,0 +1,43 @@
+<p class="govuk-body">
+  Sort by:
+  <%= govuk_link_to("Mentor (A-Z)",
+                    school_participants_path(school, sorted_by: :mentor),
+                    no_visited_state: true,
+                    classes: ["govuk-!-margin-left-2"]) %>
```

Comment:
The `classes` param isn’t working here - think this might have been a recent change to the components gem (@peteryates?). Looks like it should be `class` instead.

```suggestion
  <%= govuk_link_to("Mentor (A-Z)",
                    school_participants_path(school, sorted_by: :mentor),
                    no_visited_state: true,
                    class: "govuk-!-margin-left-2") %>
```